helpers.go

```diff
@@ -59,24 +61,34 @@ func AssertEqualAPIError(t require.TestingT, err error, expected ServerResponse)
 	return assert.Equal(t, expected.Error, errorField.String())
 }
 
-func AssertAPIErrorf(t require.TestingT, actual error, expectedCode int, expectedFormat string, a ...interface{}) bool {
+// AssertAPIErrorf check that actual API error equals expected.
+func AssertAPIErrorf(t require.TestingT, actual error, httpStatus int, code codes.Code, format string, a ...interface{}) {
```

Comment:
tbh, tests doesn't look consistent now. Why do we check code in some of tests and don't in other ones?

Reply:
I think we should do that in all tests, but not in one huge PR. Or maybe in one huge PR, but not in this PR.

Reply:
I don't think that it's necessary changes for this PR. Maybe it's better to make separate PR for these changes?